### PR TITLE
[22.04] Add slices for various media encoding libs

### PR DESCRIPTION
**Pull request by ozanmakes**
These are some useful video and image encoding libraries from main and universe repos.

### CODE REVIEW

**Comment by osener** [comment]
P.S. what is the policy for including libraries such as [libfdk-aac2](https://packages.ubuntu.com/jammy/libfdk-aac2) from `multiverse` repository?

Currently `multiverse` repo is not enabled in chisel.yaml. Is it against policy to enable this repo because of licensing reasons? In that case, what would be the suggestion? At the moment I'm running a fork of this repo to build my app but I would like to upstream all my slices and switch to the default repo.

File: slices/libvpx7.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package: libvpx7
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+    contents:
+      /usr/lib/*-linux-*/libvpx.so.7*:
```
Comment (rebornplusplus) [inline slices/libvpx7.yaml:9]: These files seem missing for `armhf`:

```
/usr/lib/arm-linux-gnueabihf/vfp/neon/libvpx.so.7
/usr/lib/arm-linux-gnueabihf/vfp/neon/libvpx.so.7.0
/usr/lib/arm-linux-gnueabihf/vfp/neon/libvpx.so.7.0.0
```

From https://packages.ubuntu.com/jammy/armhf/libvpx7/filelist.

File: slices/libx264-163.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package: libx264-163
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/lib/*-linux-*/libx264.so.163*:
```
Comment (rebornplusplus) [inline slices/libx264-163.yaml:8]: I don't think we need the wildcard at the end here, being the only file there is.

Comment (rebornplusplus) [inline slices/libx264-163.yaml:8]: We should probably include the following file too, only for `arch: i386`:

```
/usr/lib/i386-linux-gnu/i686/sse2/libx264.so.163
```

From https://packages.ubuntu.com/jammy/i386/libx264-163/filelist.

File: slices/libx265-199.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+      - libnuma1_libs
+      - libstdc++6_libs
+    contents:
+      /usr/lib/*-linux-*/libx265.so.199*:
```
Comment (rebornplusplus) [inline slices/libx265-199.yaml:11]: I don't think we need the wildcard at the end here, being the only file there is.

**Review by rebornplusplus — CHANGES_REQUESTED** [review]
Thanks for the slices! I have left a few comments.

**Comment by rebornplusplus** [comment]
> P.S. what is the policy for including libraries such as [libfdk-aac2](https://packages.ubuntu.com/jammy/libfdk-aac2) from `multiverse` repository?
> 
> Currently `multiverse` repo is not enabled in chisel.yaml. Is it against policy to enable this repo because of licensing reasons? In that case, what would be the suggestion? At the moment I'm running a fork of this repo to build my app but I would like to upstream all my slices and switch to the default repo.

Good question! ping @niemeyer @cjdcordeiro @jnsgruk

**Comment by cjdcordeiro** [comment]
> > P.S. what is the policy for including libraries such as [libfdk-aac2](https://packages.ubuntu.com/jammy/libfdk-aac2) from `multiverse` repository?
> > Currently `multiverse` repo is not enabled in chisel.yaml. Is it against policy to enable this repo because of licensing reasons? In that case, what would be the suggestion? At the moment I'm running a fork of this repo to build my app but I would like to upstream all my slices and switch to the default repo.
> 
> Good question! ping @niemeyer @cjdcordeiro @jnsgruk

No way to do that yet, unfortunately, and support for it is yet to be discussed, but at first glance, it might be something chisel would support in the future

**Comment by linostar** [comment]
Hi @ozanmakes 
Are you still interested in having those slices merged? In which case, can you please update them with the changes requested?

**Comment by rebornplusplus** [comment]
Hi @ozanmakes, as Anas noted above it has been some time since the last activity here. There are some changes requested as well. Will you please let me know if you have some time to go over the comments and update the PR? Thank you!

File: slices/libmpeg2-4.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+package: libmpeg2-4
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/lib/*-linux-*/libmpeg2.so.0*:
```
[the remark this answers is not in the document]
Comment (cjdcordeiro) [reply slices/libmpeg2-4.yaml:11]: added

File: slices/libvpx7.yaml (hunk 1 above)
Comment (cjdcordeiro) [reply slices/libvpx7.yaml:12]: added

File: slices/libx264-163.yaml (hunk 2 above)
Comment (cjdcordeiro) [reply slices/libx264-163.yaml]: y

Comment (cjdcordeiro) [reply slices/libx264-163.yaml]: added

File: slices/libx265-199.yaml (hunk 3 above)
Comment (cjdcordeiro) [reply slices/libx265-199.yaml]: y

**Review by cjdcordeiro — APPROVED** [review]
@rebornplusplus can you please have a look

**Comment by cjdcordeiro** [comment]
@ozanmakes if you sign the CLA we'd be happy to keep you as the author of these commits. 

In any case, the PR should be ready to go

**Review by linostar — APPROVED** [review]
LGTM!

**Comment by cjdcordeiro** [comment]
moving to #354

**Pull request closed without merging**